encoder_record_test.go

```diff
@@ -139,6 +139,43 @@ func TestEncoder_RecordStructPartialWithNullDefault(t *testing.T) {
 	assert.Equal(t, []byte{0x00, 0x06, 0x66, 0x6f, 0x6f}, buf.Bytes())
 }
 
+func TestEncoder_RecordStructPartialWithSubRecordDefault(t *testing.T) {
+	defer ConfigTeardown()
+
+	// register the sub record as separate schema; nested records do not support default value
```

Comment:
This comment is not correct, and functionally in the library, there is no difference between this and it being a single schema.

Reply:
You're right! I misread an old answer in stackOverflow...